http/css/style.css

```diff
@@ -65,6 +65,11 @@ p { max-width: 800px; }
 .hCell-sep       { width: 0.3em; }
 .hCell-driver    { min-width: 5.1em; }
 
+.permalink { margin-left: 0.3em; display: none; }
+.permalink:link { text-decoration: none; color: #DDDDDD; }
+.permalink:visited { text-decoration: none; color: #DDDDDD; }
+:hover > .permalink { display: inline; }
```

Comment:
If you use `display: initial;` here instead, this will work in the future if/when `.permalink` is a block element, instead of an inline element.